### PR TITLE
MRG: Update install and contrib docs

### LINKED ISSUE #3723: BUG: Clarify install instructions
We should make it clear that people should install Python2 for now, due to Mayavi.

Also, we need to make sure the instructions still work. Apparently Anaconda has been having some problems due to PyQt5 / matplotlib / mayavi junk. I suspect we'll need to add a line, at least for a little while, along the lines of `conda install numpy scipy sklearn mayavi matplotlib` to force dependency resolution. And this stuff should be backported.

Thanks @bloyl for suffering through the existing directions.

### PR DESCRIPTION
**Pull request by Eric89GXL**
Simplifies the install and contrib docs.

Closes #3723.

### CODE REVIEW

**Comment by agramfort** [comment]
can you share a link to the rendered doc?

**Comment by Eric89GXL** [comment]
For some reason CircleCI isn't saving the HTML. I even restarted it. Other PRs (e.g., #3729) are having the same problem. So you'll have to do `make clean; make html_dev-noplot; make show` locally :(

**Comment by agramfort** [comment]
:( :(

is it just on PRs?

**Comment by Eric89GXL** [comment]
Don't know, haven't checked the main builds

**Comment by agramfort** [comment]
text looks good so if you want to merge...

File: doc/install_mne_python.rst [hunk 1]
```diff
@@ -29,89 +21,74 @@ Step-by-step instructions to accomplish this are given below.
...
 .. code-block:: bash
 
-    $ ipython --matplotlib=qt
-
-Now that you have a working Python environment you can install MNE-Python.
-
-If you want to have an environment with a clean MATLAB-like interface,
-consider using Spyder_, which can easily be installed with Anaconda
-as:
+    $ conda install scipy matplotlib scikit-learn mayavi
```
Comment (christianbrodbeck) [inline doc/install_mne_python.rst]: Should we add `ipython`? When I did this on a new env, iPython was not installed, and hence `$ ipython` opened my root environment which was confusing for a moment.

File: doc/known_projects.inc [hunk 2]
```diff
@@ -49,3 +49,10 @@
 .. Astropy
 .. _Astropy: http://www.astropy.org
 .. _`Astropy GitHub`: http://github.com/astropy/astropy
+
+.. Nose
+.. _Nose: https://nose.readthedocs.io/
+
+.. Flake8
+
```
Comment (christianbrodbeck) [inline doc/known_projects.inc]: no newline?

**Comment by Eric89GXL** [comment]
> has someone looked into creating a conda channel for mne?

I haven't been motivated to do it because `pip` is so easy for people. But if it's simple and only adds small steps to a release, why not. Do you have time to try it? We plan to push out 0.14 in the next week or two if possible so it would be a good time.

**Comment by Eric89GXL** [comment]
Thanks for the look @christianbrodbeck @agramfort commit pushed. I'll merge once CircleCI comes back, hopefully with a rendered version this time.

**Comment by christianbrodbeck** [comment]
I’m currently looking into it, I will open an issue to report on it

**Comment by Eric89GXL** [comment]
@christianbrodbeck there is already a bit on the wiki that could be a good starting point:

https://github.com/mne-tools/mne-python/wiki/How-to-make-a-release

**Pull request merged**